src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -98,9 +98,13 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
 
   const handleManualFailureRateUpdate = (eventType: EventType) => {
     if (eventType === EventType.EXTERNAL) {
-      onEventUpdated({ ...shapeToolData, probability: externalManuallyDefinedFailureRate });
+      onEventUpdated({
+        ...shapeToolData,
+        probability: externalManuallyDefinedFailureRate,
```

Comment:
put it to a separate variable as you are calling

Reply:
Refactored.